recipe_scrapers/projectgezond.py

```diff
@@ -0,0 +1,96 @@
+# mypy: allow-untyped-defs
+import re
+
+from ._abstract import AbstractScraper
+
+
+class ProjectGezond(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "projectgezond.nl"
+
+    def author(self):
+        return "Project Gezond"
+
+    def title(self):
+        return self.soup.find("h1", {"class": "entry-title"}).text
+
+    def category(self):
+        return ", ".join(
+            [
+                element.text
+                for element in self.soup.find(
+                    "span", {"class": "meta-category"}
+                ).find_all(
+                    "a", {"class": lambda x: x is not None and x.startswith("category")}
+                )
+            ]
+        )
+
+    def total_time(self):
+        time_element = self.soup.find("em", string="Bereidingstijd:").parent
+        return "".join(
+            [
+                element.text
+                for element in time_element.children
+                if element.text != "Bereidingstijd:"
+            ]
+        ).strip()
+
+    def yields(self):
+        # Match everything in the h2 with 'Dit heb je nodig'
+        # The text inside the parentheses contains the yield for the ingredients that are listed
+        return re.search(
```

Comment:
One more thing to keep in mind - hopefully not important in this case, but in general - and sorry if I'm explaining things that you understand already, but it's worth being careful to limit what regular expressions can match on, and/or how much input text they are provided as input.

Just something I repeat (no pun intended) at nearly every available opportunity :smile: